apps/package.json

```diff
@@ -139,7 +139,7 @@
     "node-sass": "^4.0.0",
     "npm-which": "2.0.0",
     "phantomjs-prebuilt": "^2.1.3",
-    "playground-io": "0.4.0",
+    "playground-io": "code-dot-org/playground-io",
```

Comment:
it'd be nice to pin to a version number on our fork, particularly since it seems like we're going to be doing this semi-regularly